### PR TITLE
CCU-277 fix participant status (lowercase instead of uppercase)

### PR DESCRIPTION
**Pull request by lerami**

### CODE REVIEW

File: src/app/reducers/ParticipantsWaitingReducer.js [hunk 1]
```diff
@@ -57,7 +56,7 @@ const ParticipantsWaitingReducer = (state = defaultState, action) => {
       participants[index].status = action.payload.status;
       participants[index].type = action.payload.type;
       participants[index].isConnected =
-        action.payload.status == "Connected" ? true : false;
+        action.payload.status = STATUS_CONNECTED ? true : false;
```
Comment (t-gilb) [inline src/app/reducers/ParticipantsWaitingReducer.js:59]: could be simplified:

status = action.payload.status === STATUS_CONNECTED;

File: src/app/reducers/ParticipantsReducer.js [hunk 2]
```diff
@@ -178,7 +178,7 @@ const ParticipantReducer = (state = defaultState, action) => {
             type: userInfo.type,
             metadata: userInfo.metadata,
             isAdmin: userInfo.metadata?.admin === "true",
-            isConnected: userInfo.status == "Connected" ? true : false,
+            isConnected: userInfo.status == STATUS_CONNECTED ? true : false,
```
Comment (t-gilb) [inline src/app/reducers/ParticipantsReducer.js:181]: === instead of == 
see type coercion

Comment (lerami) [reply src/app/reducers/ParticipantsReducer.js]: fixed

File: src/app/reducers/ParticipantsWaitingReducer.js (hunk 1 above)
Comment (lerami) [reply src/app/reducers/ParticipantsWaitingReducer.js]: fixed

**Review by t-gilb — APPROVED** [review]

**Pull request merged**